docs/generator.md

```diff
@@ -33,6 +33,25 @@ const output = generate(
 The symbols like white spaces or new line characters are not preserved in the AST. When Babel generator prints code from the AST, the output format is not guaranteed.
 :::
 
+### Parser plugins support
+Babel generator supports all the listed [Babel parser plugins](./parser.md#plugins) except `estree`. Note that parser plugins do not transform the code. For example,
```

Comment:
Do we support the `placeholders` plugin in the generator?

Reply:
Yes.

https://github.com/babel/babel/blob/dba45d3ebc92a868c9b8952a6f407c6ab83c6b3f/packages/babel-generator/src/generators/base.ts#L98-L106